server/src/main/java/com/server/loan/calculator/services/CalculatorService.java

```diff
@@ -0,0 +1,60 @@
+package com.server.loan.calculator.services;
+
+import com.server.loan.calculator.exception.ValueOutOfRangeException;
+import com.server.loan.calculator.models.CalculatorData;
+import com.server.loan.calculator.validators.InputAsCorrectNumberValidator;
+import com.server.loan.calculator.validators.ValueInRangeValidator;
+import lombok.Data;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+@Service
+@Data
+public class CalculatorService {
+
+    private final CalculatorData data;
```

Comment:
should not be auto injected, later you will instantiate this class through endpoint controller or database DAO, so in this case you should just do new CalculatorData()